### ISSUE TITLE
toolbar.html.twig overrides core classes

### ISSUE THREAD

**Issue by rutiolma** [openfed/kiso#37]
With the new Openfed version, we'll have to drop toolbar_themes module, because it is not compatible with Drupal9 and is poorly maintained.
As such, we'll start using Drupal core Toolbar and its template.
I see that Kiso overrides this template (templates/toolbar.html.twig), maybe to make it more accessible, but it also overrides important classes, and without them the style is broken.

Since templates/toolbar.html.twig is based on core toolbar.html.twig can you make that any "toolbar_*" class is converted to "toolbar-*" like in core: https://api.drupal.org/api/drupal/core%21modules%21toolbar%21templates%21toolbar.html.twig/8.9.x

I don't see a reason to override these classes.

**Comment by smillart** [comment]
Indeed, I made some markup improvements regarding unexpected `<h2-3>` heading elements which are used in the core `toolbar.html.twig` template but appear before the `<h1>` first-level heading element giving the document a wrong structure for _Assistive Technologies (AT)_.

Concerning the core classes override, Kiso stylesheets are based on the [Drupal CSS coding standards and best practices](https://www.drupal.org/docs/develop/standards/css/css-architecture-for-drupal-8#formatting) ([BEM Methodology](https://en.bem.info/methodology/)) regarding the formatting of class names:

```css
/* BEM for Block, Element & Modifier */
.block { ... }
.block__element { ... }
.block__element--modifier

/* Administrative toolbar class names formatting */
.toolbar { ... }
.toolbar__bar { ... }
.toolbar__tab { ... }
.toolbar-menu { ... }
.toolbar-menu__item { ... }
.toolbar-menu__item--expanded { ... }
.toolbar-menu__item--collapsed { ... }
```

However, I noticed that I didn't create the stylesheet which is supposed to apply the Kiso "Lool & Feel". I gonna fix this oversight! :-\

**Comment by smillart** [comment]
Regarding the overridden classes, I gonna revert to the original ones for this issue in priority. For a next release, I will upgrade them in the following issue #46 to meet the Kiso stylesheets guidelines.

**Comment by smillart** [comment]
This enhancement will be available with the next **release 2.7**.
